app/src/core/player.ts

```diff
@@ -132,7 +136,7 @@ export class Player {
   ): void {
     const lastRenderItem = this.renderItems[this.renderItems.length - 1];
     const callback = () => {
-      const time = getTime();
+      const time = Math.min(getTime(), renderItem.absoluteStart + renderItem.length);
```

Comment:
I don't understand what's going on here

Reply:
sometimes the player is stopped a bit too late and the time is advanced to much. this prevents that

Reply:
added a comment explaining it